corehq/apps/app_manager/static/app_manager/js/xpathConfig.js

```diff
@@ -0,0 +1 @@
+../../../xpath_validator/nodejs/xpathConfig.js
```

Comment:
haha that's pretty nifty. is this how github diffs symlinks? didn't even know you could commit those. pretty cool if so. 

@snopoke this may be a way we can commit a symlink to the ansible directory to have ansible vault pre hook commits work automatically